### DISCUSSION TITLE
Fighting Stance - Unarmed Skill - Suplexing/Throwing people

### DISCUSSION THREAD

**Discussion by terrykwan1** [FakeFishGames/Barotrauma#6359] [Ideas and suggestions]
Barotrauma already has limb based damage, ragdoll physics and a built-in character editor. It could also be a fighting game.

Switch to unarmed fighting stance by pressing ALT.

In this stance holding right click lets u block with your arms, which lets u use them as a sort of shield to protect your vitals and head.

Left click punches instead of interacting with doors and stuff.

Gain ability to jump slightly.

In fighting stance G for grab is more violent, so u can grab and suplex people with certain inputs. G initiates a combat grab where giving a left/right input throws the target a down input performs a suplex on the target.

This doesn't have to be instant, there can be like a struggle meter or something when somebody gets grabbed that is modified by their own unarmed which is their ability to resist being grabbed/thrown.

Unarmed skill greatly effects the force of the throw and damage/stun of suplexing people.

----

Enemies can also now grab you in their jaws to drag people away or in circles, this is resisted by unarmed.

**Comment by ghost** [comment]
I think adding blocking and unarmed punches would be a good idea. 

I don't agree with this being how they work however, so the versions I'd like to suggest are:
Blocking: Hold right click with nothing in your hands to use your arms to cover your face, allowing hitboxes to protect your head and let the arms take hits instead.

Punching: Left click while blocking to punch, deals a decent amount of blunt force trauma to an enemy, but also deals some to yourself if you hit armour or a diving suit. (Yes, punching metal can hurt!).
